marshmallow/fields.py

```diff
@@ -1016,32 +1027,86 @@ def _deserialize(self, value, attr, data):
 class Date(Field):
     """ISO8601-formatted date string.
 
+    :param strict: If True, only accept Date objets. If False, will accept any
+        object that has integer ``year``, ``month`` and ``day`` values.
+        Default False.
+    :param format: Either ``"iso"`` (for ISO8601) or a date format string.
+        If `None`, defaults to "iso".
     :param kwargs: The same keyword arguments that :class:`Field` receives.
     """
     default_error_messages = {
         'invalid': 'Not a valid date.',
         'format': '"{input}" cannot be formatted as a date.',
     }
 
-    def _serialize(self, value, attr, obj):
-        if value is None:
-            return None
+    DATEFORMAT_SERIALIZATION_FUNCS = {
+        'iso': utils.to_iso_date,
+        'iso8601': utils.to_iso_date,
+    }
+
+    DATEFORMAT_DESERIALIZATION_FUNCS = {
+        'iso': utils.from_iso_date,
+        'iso8601': utils.from_iso_date,
+    }
+
+    DEFAULT_FORMAT = 'iso'
+
+    def __init__(self, strict=False, format=None, **kwargs):
+        super(Date, self).__init__(**kwargs)
+        self.strict = strict
+        self.dateformat = format
```

Comment:
It might be better to avoid field name specific prefixes to make these properties nicer to override with inheritance. They are already scoped under a class, so there is no benefit to further namespacing them.

- `self.dateformat` -> `self.format`
- `DATEFORMAT_SERIALIZATION_FUNCS` -> `SERIALIZATION_FUNCS`
- `DATEFORMAT_DESERIALIZATION_FUNCS ` -> `DESERIALIZATION_FUNCS `

Reply:
That makes sense to me. I'll make the changes.